tripy/tripy/frontend/trace/ops/storage.py

```diff
@@ -79,7 +79,7 @@ def str_skip_fields(self) -> Set[str]:
             return {"data"}
         return set()
 
-    def __eq__(self, other) -> bool:
+    def __eq__(self, other: "tripy.Storage") -> bool:
```

Comment:
Is this being added to the function registry or are you just adding the type annotation for readability? (It shouldn't be part of the function registry)

Reply:
It was in the registry due to export.public_api and it gave an error without it.

Reply:
Why is this being exported? `Storage` is just a trace op, not a user-facing API.

Reply:
Looks like I was mistaken. I was getting errors from the registry and saw this listed in the stack trace when I was debugging, but it's indeed not exported and it did not cause errors to remove it.